opdracht-1/selectors.css

```diff
@@ -4,32 +4,57 @@
 
 /* 1.1 Geef de 'img' tag in the header een ID en zet zijn breedte op 100% */
 
-
+#fist_header{
+    width: 100%;
+}
 
 /* 1.2 Verander de kleur van alle list items in de unordered list naar hsl(110, 29%, 52%) */
 
-
+ul{
+    color: hsl(110, 29%, 52%);
+}
 
 /* 1.3 Geef de 'span' tag in de footer een beschrijvende class name en maak de lettergrootte kleiner  */
 
-
+.site_info{
+    font-size: 10px;
+}
 
 /* 1.4 Haal de text-decoration van navigatie-links weg, geef onbezochte links de kleur hsl(359, 79%, 56%) en verander de kleur wanneer de gebruiker er overheen hovert. */
 
+nav a{
+    text-decoration: none;
+}
 
+nav a:link{
+    color: hsl(359, 79%, 56%);
+}
 
+nav a:hover{
+    color: green;
+}
 /* 1.5 Maak een hover- en visited stijl voor alle links in 'main'. */
 
+main a:link{
+    color: white;
+}
 
+main a:hover{
+    color: aqua;
+}
 
 /* Verander de achtergrond van de SignUp button. Tip: https://developer.mozilla.org/en-US/docs/Web/CSS/Attribute_selectors  */
 
-
+.subscribe input:last-child{
+    background: green;
+}
 
 /* Geef het tekst-veld een blauwe rand wanneer hij in focus is */
 
```

Comment:
Hier had je het textveld een blauwe rand moeten geven wanneer de cursor op het veld klikt (in focus).
Ik heb dit op de volgende manier opgeslost:

div input[type=text]:focus{
    outline-color: blue;
}